packages/shared/components/modals/FilterModal.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="typescript">
     import { FontWeight } from 'shared/components/Text.svelte'
     import { Modal, Text } from 'shared/components'
-    import { Filter } from '@core/wallet/interfaces/filter.interface'
+    import type { Filter } from '@core/wallet/interfaces/filter.interface'
```

Comment:
Is there a reason why this import path is so specific?

Reply:
No, this was probably auto-filled by VS Code. I'll change it.